packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Internal
+
+-   Remove usage of deprecated spreading of `key` prop in JSX ([#61692](https://github.com/WordPress/gutenberg/pull/61692)).
```

Comment:
For better reference it might be worth including the component names it occurred in.

Reply:
Updated.